aws.mk

```diff
@@ -1,18 +1,43 @@
-include $(shell git rev-parse --show-toplevel)/terraform.mk
```

Comment:
ah ok. so multiple makefiles are being parsed from `terraform.mk` as the top source of ultimate configurable truth :tm: ?